ui/raidboss/data/06-ew/raid/p9s.ts

```diff
@@ -235,7 +268,109 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'P9S Limit Cut',
+      // Ball of Levin combatants are added ~0.3 seconds after Kokytos finishes using Levinstrike Summoning
+      // and ~1.7 before Kokytos begins using Scrambled Succession (which is when limit cut markers appear)
+      // These combatants are added in their actual positions, so no need to check OP for combatant data.
+      id: 'P9S Limit Cut Levin Orb Collect',
+      type: 'AddedCombatant',
+      // There are multiple invsible combatants that share this name, but the ones that receive HeadMarkers
+      // in limit cut (the ones we care about) are distinguishable because their level attribute is set to 90.
+      netRegex: { name: 'Ball of Levin', level: '5A' },
+      run: (data, matches) => {
+        // (0 = N, 1 = NE ... 7 = NW)
+        const orb8Dir = Directions.addedCombatantPosTo8Dir(matches, centerX, centerY);
+
+        // Orbs should always spawn on cardinals
```

Comment:
This is not true.  They can also be on intercards.

Reply:
Haha, no worries, I was just literally reviewing a vod from last week and saw it was intercards.  There were so many savage things I saw last week where it was like "oh yeah we looked at 6 videos and it was always X (spoilers: it is not always X)".

(Also sorry sorry if that last comment was curt!  I wasn't trying to be rude.  I super appreciate this PR, and there's nothing to be embarrassed about.)